fe/fe-core/src/main/java/com/starrocks/scheduler/TaskRunExecutor.java

```diff
@@ -0,0 +1,53 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.scheduler;
+
+import com.starrocks.statistic.Constants;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.Future;
+
+
+public class TaskRunExecutor {
+    private static final Logger LOG = LogManager.getLogger(TaskRunExecutor.class);
+    private final ExecutorService taskRunPool = Executors.newCachedThreadPool();
```

Comment:
I suggest you add a configuration to limit the usage of thread pool.

Reply:
Yes, these features will be added in later PRs.